pep-0484.txt

```diff
@@ -1063,6 +1063,54 @@ in turn, to ``collections.abc.Callable``::
       ...
 
 
+The ``NoReturn`` type
+---------------------
+
+The ``typing`` module provides a special type ``NoReturn`` to annotate functions
+that never return normally. For example, a function that unconditionally
+raises an exception::
+
+  from typing import NoReturn
+
+  def stop() -> NoReturn:
+      raise Exception('no way')
+
+Such return annotation is used in ``typeshed`` stubs for functions
```

Comment:
Such return -> The ``NoReturn`` (mostly to avoid using "such" twice in one sentence :-).

Also, drop "in typeshed".